frontend/pages/product_new_page.js

```diff
@@ -108,6 +112,21 @@ export default class AddProductForm extends PureComponent {
           <h2 className='page-title'>
             Add a new product
           </h2>
+          <div className='dropdown-data-input-status'>
+            <FormSelect
+              options={[
+                { value: productStatusIncomplete.valuel, label: productStatusIncomplete.value },
```

Comment:
you're setting both the value and label to `productStatusIncomplete.value`. Also, you dont need any of this, you can just import `getProductStatusSelectOptions` and then do this:

```
<FormSelect
  options={getProductStatusSelectOptions()}
  ... />
```